src/integration-tests/introspector/introspectTest.sh

```diff
@@ -26,7 +26,16 @@
 #   IMAGE_NAME, etc, all have defaults, or can be passed in.  See the 'export'
 #   calls in the implementation below for the complete list.
 #
-
+# Usage:
```

Comment:
Please update README file with this information too.

Reply:
done